### ISSUE TITLE
new  update linux-zen not restart

### ISSUE THREAD

**Issue by medioman** [zen-kernel/zen-kernel#130]
Hello, this morning I updated the kernel-zen and my arch was not able to restart, it starts in shell recovery.
He does not find the uuid of the partizone.
If I boot with the second kernel and do downgrade of linux-zen the machine starts again.
thank you

**Comment by damentz** [comment]
@medioman are you referring to version "4.18.11.zen1-1" of linux-zen?

Looking at the changes, the configuration was updated to make particular USB configurations.

https://git.archlinux.org/svntogit/packages.git/commit/trunk?h=packages/linux-zen&id=b55617296b6ee842fd5962525f3feeccc7752e0a

Also if you can, please provide the output of `inxi -b`, and since this is regarding finding your disk, the output of `inxi -d`.  If you're using arch, the package you need to install is `inxi`, it should be available from the main repos: https://aur.archlinux.org/packages/inxi/

**Comment by medioman** [comment]
[don@don-pc ~]$ inxi -du
Drives:
  Local Storage: total: 931.52 GiB used: 404.93 GiB (43.5%) 
  ID-1: /dev/sda type: USB vendor: Samsung model: Portable SSD T1 250GB 
  size: 232.89 GiB 
  ID-2: /dev/sdb vendor: Toshiba model: MQ01ABD075 size: 698.64 GiB 
  Optical-1: /dev/sr0 vendor: hp model: DVD-RAM UJ8D1 dev-links: cdrom 
  Features: speed: 24 multisession: yes audio: yes dvd: yes 
  rw: cd-r,cd-rw,dvd-r,dvd-ram 
Partition:
  ID-1: / size: 17.91 GiB used: 13.99 GiB (78.1%) fs: ext4 dev: /dev/sda8 
  uuid: 91a7796a-ed61-4963-9026-4456b3389029 
[don@don-pc ~]$ inxi -b
System:
  Host: don-pc Kernel: 4.18.10-zen1-1-zen x86_64 bits: 64 
  Desktop: Openbox 3.6.1 Distro: Arch Linux 
Machine:
  Type: Laptop System: Hewlett-Packard product: HP ENVY dv7 Notebook PC 
  v: 088A120000305910000620100 serial: <root required> 
  Mobo: Hewlett-Packard model: 181D v: 52.24 serial: <root required> 
  UEFI: Insyde v: F.2E date: 09/07/2016 
CPU:
  Quad Core: Intel Core i7-3630QM type: MT MCP speed: 1197 MHz 
  min/max: 1200/2401 MHz 
Graphics:
  Device-1: Intel 3rd Gen Core processor Graphics driver: i915 v: kernel 
  Device-2: NVIDIA GF108M [GeForce GT 635M] driver: nouveau v: kernel 
  Display: x11 server: X.Org 1.20.1 driver: intel,nouveau 
  unloaded: fbdev,modesetting,vesa resolution: 1600x900~60Hz 
  OpenGL: renderer: Mesa DRI Intel Ivybridge Mobile v: 4.2 Mesa 18.2.1 
Network:
  Device-1: Qualcomm Atheros AR9285 Wireless Network Adapter driver: ath9k 
  Device-2: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet 
  driver: r8169 
Drives:
  Local Storage: total: 931.52 GiB used: 404.93 GiB (43.5%) 
Info:
  Processes: 200 Uptime: 2h 45m Memory: 7.68 GiB used: 1.03 GiB (13.4%) 
  Shell: bash inxi: 3.0.24 
[don@don-pc ~]$

**Comment by medioman** [comment]
yes, the distro is installed on an external hard drive, which I have to do, know that I'm not a great expert, thanks for help

**Comment by damentz** [comment]
Per new issue #131, here are comments from @mdeioman 

Screenshot: https://scrot.moe/image/9Wa3p

inxi output:

```
[don@don-pc ~]$ inxi -du
Drives:
Local Storage: total: 943.03 GiB used: 17.63 GiB (1.9%)
ID-1: /dev/sda type: USB vendor: Samsung model: Portable SSD T1 250GB
size: 232.89 GiB
ID-2: /dev/sdb vendor: Toshiba model: MQ01ABD075 size: 698.64 GiB
ID-3: /dev/sdc type: USB vendor: Samsung model: File-Stor Gadget
size: 11.51 GiB
Optical-1: /dev/sr0 vendor: hp model: DVD-RAM UJ8D1 dev-links: cdrom
Features: speed: 24 multisession: yes audio: yes dvd: yes
rw: cd-r,cd-rw,dvd-r,dvd-ram
Partition:
ID-1: / size: 17.91 GiB used: 14.17 GiB (79.1%) fs: ext4 dev: /dev/sda8
uuid: 91a7796a-ed61-4963-9026-4456b3389029
[don@don-pc ~]$
```

```
[don@don-pc ~]$ inxi -b
System:
  Host: don-pc Kernel: 4.8.14-1-ARCH x86_64 bits: 64 Desktop: Openbox 3.6.1 
  Distro: Arch Linux 
Machine:
  Type: Laptop System: Hewlett-Packard product: HP ENVY dv7 Notebook PC 
  v: 088A120000305910000620100 serial: <root required> 
  Mobo: Hewlett-Packard model: 181D v: 52.24 serial: <root required> 
  UEFI: Insyde v: F.2E date: 09/07/2016 
CPU:
  Quad Core: Intel Core i7-3630QM type: MT MCP speed: 1200 MHz 
  min/max: 1200/2401 MHz 
Graphics:
  Device-1: Intel 3rd Gen Core processor Graphics driver: i915 v: kernel 
  Device-2: NVIDIA GF108M [GeForce GT 635M] driver: nouveau v: kernel 
  Display: x11 server: X.Org 1.20.1 driver: intel,nouveau 
  unloaded: fbdev,modesetting,vesa resolution: 1600x900~60Hz 
  OpenGL: renderer: Mesa DRI Intel Ivybridge Mobile v: 4.2 Mesa 18.2.1 
Network:
  Device-1: Qualcomm Atheros AR9285 Wireless Network Adapter driver: ath9k 
  Device-2: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet 
  driver: r8169 
Drives:
  Local Storage: total: 943.03 GiB used: 17.63 GiB (1.9%) 
Info:
  Processes: 183 Uptime: 28m Memory: 7.69 GiB used: 1.30 GiB (16.9%) 
  Shell: bash inxi: 3.0.24 
[don@don-pc ~]$ 
```

Also, @medioman mentioned his installation is on an external drive.

**Comment by damentz** [comment]
@heftig looks like this is related to the config changes and static USB modules
